documentation/file_structure.md

```diff
@@ -15,19 +15,30 @@ Directory | Description
 Directory | Description
 :---------|:------------
 /arch	  | Architecture and platform specific files
+/include  | Header files of resources exported in the core
 /lib	  | Generic libraries that are likely to be replaced in a final product
 /mm	  | Generic memory management, currently empty
 /tee	  | Generic TEE files
 
+## Structure of /core/include
+Directory | Description
+:---------|:------------
+/crypto	  | Include files expoising API for /core/crypto files
+/drivers  | Include files expoising API for /core/drivers files
+/dt-bindings  | Include files for the device tree bindings
```

Comment:
Speaking of bindings... they have to be documented somewhere. Do you expect to introduce (secure) DT stuff that would not be documented upstream in the Linux kernel?

Reply:
No, all the bindings are expected generic whatever the secure state of the related device.
The secure part of it will come through use of the `secure-` prefix. We currently plan to use it only over the `status` property.

Reply:
Sounds good